.github/workflows/run-metrics.yaml

```diff
@@ -21,10 +20,6 @@ jobs:
         uses: actions/download-artifact@v3
         with:
           name: kata-static-tarball-amd64${{ inputs.tarball-suffix }}
-          path: ${{ ARTIFACTS_DIR }}
-
-      - name: Install kata
-        run: bash tests/metrics/gha-run.sh install-kata ${{ ARTIFACTS_DIR }}
```

Comment:
could you please replace the variable name 'ARTIFACTS_DIR ' with the value on that variable?
`run: bash tests/metrics/gha-run.sh install-kata kata-artifacts`